pkg/services/ngalert/notifier/alertmanager.go

```diff
@@ -44,7 +44,7 @@ type AlertingStore interface {
 	store.ImageStore
 }
 
-type Alertmanager struct {
+type alertmanager struct {
```

Comment:
Here we're changing the struct to a package-private struct, but with package-public methods? Is that what we want?